src/features/controls.ts

```diff
@@ -385,7 +390,8 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
 
         const maximumFieldOfView = this[$framedFieldOfView]!;
         controls.applyOptions({maximumFieldOfView});
-        controls.setFieldOfView(zoom * this[$framedFieldOfView]!);
+        // TODO(#835): Move computation of this value to Model or ModelScene
+        this[$idealFieldOfView] = this[$framedFieldOfView]! * zoom;
```

Comment:
I think I might call this $currentFieldOfView, since zoom is coming from user interaction. $framedFieldOfView is as ideal as it gets.

Reply:
But this is only applied when the `field-of-view` is set to `auto`. In theory, this number should not vary for a given model and aspect ratio, right?

Reply:
$framedFieldOfView is exactly as you've described. $idealFieldOfView is here based on zoom, which is dependent on how the user has zoomed, which makes it no longer ideal.

Reply:
Okay, I may have misunderstood what was happening here then. I'll try a different approach.

Reply:
I changed some of the names. I *think* the logic is sound, and comparable to what we had before, but I would appreciate any feedback that suggests otherwise!